packages/apps/src/Menu/ChainInfo.tsx

```diff
@@ -91,16 +91,24 @@ export default React.memo(styled(ChainInfo)`
     .info {
       flex: 1;
       padding-right: 0.5rem;
-
-      .bestNumber,
+      text-align: left;
+ 
       .chain {
-        font-size: 0.9rem;
-        line-height: 1.2;
+        max-width: 9rem;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        font-weight: 600;
+        font-size: 1.15rem;
+        line-height: 1.5rem;
+        color: #FFFFFF;
```

Comment:
This has exceptionally weird side-effects. (as the the chain name). The issue is that the font color adapts to the background, on dark it is white on light it is dark gray.

So what happens here now is that as we swap chains, the color changes from being aligned to not being aligned.

(See for instance Amber vs Cantilon testnets)

Reply:
Example (the white here is very wrong for the light background) -

![image](https://user-images.githubusercontent.com/1424473/103935456-e35c0d80-5126-11eb-8df0-1a650270d6d5.png)

UI inconsistent with the above just after swapping, above different, this the same -

![image](https://user-images.githubusercontent.com/1424473/103935555-0dadcb00-5127-11eb-8c24-e8a3ffa9faea.png)